finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,24 @@
+var wordedNums = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 'ten'];
+for (i = 10; i > 0; i--) {
+  newFrogs = i - 1;
+  newFrogWorded = wordedNums[newFrogs]; //only works for line 4
+  frogWorded = wordedNums[i]; //only works for line 1 as it does not decrement until loop starts again
+  if(newFrogs === 0){
+    console.log(frogWorded + ' speckled frogs sat on a log');
```

Comment:
Instead of having to `console.log` here several times, you can also use the new line character `\n` to force a new line within the string.